Marlin/src/HAL/HAL_ESP32/HAL_timers_ESP32.cpp

```diff
@@ -132,7 +132,7 @@ void HAL_timer_start(const uint8_t timer_num, uint32_t frequency) {
   timer_enable_intr(timer.group, timer.idx);
 
   // TODO need to deal with timer_group1_isr
-  timer_isr_register(timer.group, timer.idx, timer_group0_isr, (void*)timer.idx, NULL, NULL);
+  timer_isr_register(timer.group, timer.idx, timer_group0_isr, (void*)timer.idx, nullptr, nullptr);
```

Comment:
```c++
timer_isr_register(timer.group, timer.idx, timer_group0_isr, (void*)timer.idx, ESP_INTR_FLAG_INTRDISABLED, nullptr);
```
I'm not sure, but should not the FLAG be applied here?

Reply:
Probably! I'll look closer….